frontends/article/article-page.tsx

```diff
@@ -76,26 +69,34 @@ export const Article: FC<IArticleProps> = (props) => {
     [blocks],
   );
 
+  useEffect(() => {
```

Comment:
![Screenshot 2023-02-20 at 17 11 08](https://user-images.githubusercontent.com/55176773/220102839-efe0ed6a-6bd2-46a6-ac57-774f5884e9b9.png)

You are reverting the changes you did, but NOT in right way. The reverted changes should not be here. Revert it in the way which it is not shown here. Othervise it will be very complicated to review